src/raftstore/store/store.rs

```diff
@@ -1217,6 +1276,50 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.register_split_region_check_tick(event_loop);
     }
 
+    fn register_consistency_check_tick(&self, event_loop: &mut EventLoop<Self>) {
+        if let Err(e) = register_timer(event_loop,
+                                       Tick::ConsistencyCheck,
+                                       self.cfg.consistency_check_tick_interval * 1000) {
+            error!("{} register consistency check tick err: {:?}", self.tag, e);
+        };
+    }
+
+    fn on_consistency_check_tick(&mut self, event_loop: &mut EventLoop<Self>) {
+        // To avoid frequent scan, we only check some of the regions very tick.
+        let leader_count = self.region_peers.values().filter(|p| p.is_leader()).count() as f64;
+        let capacity = cmp::max((leader_count * DEFAULT_CONSISTENCY_CHECK_PERCENT) as usize,
```

Comment:
If we have 1000 leaders, the capacity is 200, checking 200 regions may block the split/merge flow. Do we need using another thread to do the check?

/cc @ngaut